front/src/components/molecules/TotalInfoContainer/TotalInfoContainer.tsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import Grid from '@material-ui/core/Grid';
+import { styled } from '@material-ui/core/styles';
+import Tooltip from '@material-ui/core/Tooltip';
+import Avatar from '../../atoms/Avatar/Avatar';
+import Typo from '../../atoms/Typo/Typo';
+import { TotalInfoContainerProps } from '../../../interfaces/molecules/TotalInfoContainer/ITotalInfoContainer';
+import { GameHistory } from '../../../interfaces/utils/data/IUser';
+
+const RootContainer = styled(Grid)({
+  width: '100%',
+  justifyContent: 'center',
+  alignItems: 'center',
+  backgroundColor: '#f5f6fa',
+  borderRadius: '10px',
+  cursor: 'pointer',
+});
+
+const ItemContainer = styled(Grid)({
+  paddingLeft: 10,
+});
+
+const Online = styled(Grid)({
+  color: '#50CB93',
+});
+
+const Offline = styled(Grid)({
+  color: '#B2B1B9',
+});
+
+const OnlineGrid = (): React.ReactElement => (
+  <>
+    <Tooltip title="접속상태" arrow>
+      <Online>
+        <Typo variant="h6">online</Typo>
+      </Online>
+    </Tooltip>
+  </>
+);
+
+const OfflineGrid = (): React.ReactElement => (
+  <>
+    <Tooltip title="접속상태" arrow>
+      <Offline>
+        <Typo variant="h6">offline</Typo>
+      </Offline>
+    </Tooltip>
+  </>
+);
+
+const NameGrid = ({ name }: { name: string }): React.ReactElement => (
+  <>
+    <Tooltip title="이름" arrow>
+      <Grid item>
+        <Typo variant="h6">{name}</Typo>
+      </Grid>
+    </Tooltip>
+  </>
+);
+
+const ScoreGrid = ({ score }: { score: number }): React.ReactElement => (
+  <>
+    <Tooltip title="점수" arrow>
+      <Grid item>
+        <Typo variant="h6">{`${score}점`}</Typo>
+      </Grid>
+    </Tooltip>
+  </>
+);
+
+const GameHistoryGrid = ({ gameHistory }: { gameHistory: GameHistory }): React.ReactElement => {
+  const { win, lose } = gameHistory;
+
+  return (
+    <>
+      <Tooltip title="승부결과" arrow>
+        <Grid item>
+          <Typo variant="h6">{`${win}승 / ${lose}패`}</Typo>
+        </Grid>
+      </Tooltip>
+    </>
+  );
+};
+
+const TotalInfoContainer = ({
```

Comment:
```suggestion
const UserInfoContainer = ({
```
Total 이라는 용어가 현재 컨테이너에 적절하지 않습니다. 유저라는 뜻을 변수명에 넣으면 어떨까요?

Reply:
좋은것같습니다.